easy_profiler_core/CMakeLists.txt

```diff
@@ -202,8 +206,11 @@ easy_define_target_option(easy_profiler EASY_OPTION_PROFILE_SELF EASY_OPTION_MEA
 easy_define_target_option(easy_profiler EASY_OPTION_PROFILE_SELF_BLOCKS_ON EASY_OPTION_STORAGE_EXPAND_BLOCKS_ON)
 easy_define_target_option(easy_profiler EASY_OPTION_IMPLICIT_THREAD_REGISTRATION EASY_OPTION_IMPLICIT_THREAD_REGISTRATION)
 if (WIN32)
-    easy_define_target_option(easy_profiler EASY_OPTION_EVENT_TRACING EASY_OPTION_EVENT_TRACING_ENABLED)
-    easy_define_target_option(easy_profiler EASY_OPTION_LOW_PRIORITY_EVENT_TRACING EASY_OPTION_LOW_PRIORITY_EVENT_TRACING)
+    if (${CMAKE_SYSTEM_NAME} STREQUAL "WindowsStore")
```

Comment:
instead of having an empty block maybe it should be something like `if (NOT ${CMAKE_SYSTEM_NAME} STREQUAL "WindowsStore")` and no `else()`

Reply:
I actually couldn't get the `NOT` to work but I'll give it another go, maybe I was doing `NOT STREQUAL`